### DISCUSSION TITLE
Cannot apply to github student developer pack

### DISCUSSION THREAD

**Discussion by Maksewsha** [community/community#38935] [Projects and Issues]
### Select Topic Area

Bug

### Body

My university email is valid and my user id is until august 2023 and it is at the screen but i get the error:
Your application cannot be reviewed until you fix the following:
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

**Comment by katrinmisel** [comment]
For the "short bio" error, for some reason it has something to do with having access to your location. I updated my bio and my name and I was still getting the same error for 2 days straight with Chrome but when I tried to upload my proof of enrollment over the phone browser just now it worked instantly.

**Comment by iraqark** [comment]
same
